open_humans/management/commands/suspend_users.py

```diff
@@ -44,7 +53,40 @@ def handle(self, *args, **options):
                     raise CommandError('User ID "{}" does not exist!'
                                        .format(id_str))
 
+        if options['email']:
+            self.email_notification(users_to_suspend)
```

Comment:
Do I see this correctly that the email notifications about suspension go out even if `skip_suspend` is active and no actual suspension is performed?

Reply:
If `--email --skip-suspend` is used then yes.

I added this because I was concerned about a scenario where suspension had been done but the `--email` flag was forgotten, in which case this would be the correction... but... maybe that makes no sense, it's harmless to go ahead with suspend?

Reply:
Ah, sorry. I had missed the email flag, making that optional too, so one can recover from both mistakes (sending email, not suspending & vice versa). Great! 🙂

Reply:
nah, on reflection your question makes it clear that "skip-suspend" is unnecessary, it's harmless to "re-suspend" users to redo the command to send emails. so I've removed it, thanks!

Reply:
ok! 👍